moped-database/metadata/tables.yaml

```diff
@@ -4072,31 +4072,47 @@
         insert:
           columns: '*'
         update:
-          columns:
-            - is_deleted
-            - is_scanned
-            - created_by
-            - file_size
-            - file_type
-            - project_file_id
-            - project_id
-            - api_response
-            - file_metadata
-            - file_permissions
-            - file_description
-            - file_key
-            - file_name
-            - create_date
+          columns: '*'
```

Comment:
Is this a new `graphql-engine` syntax? If so, I'm really happy to see it! `*` can expand with new columns without manually having to specify them, which I think is a good fit for a lot of what we do and also very within the spirit of Hasura's `graphql-engine` philosophy.

Reply:
agreed! it's super useful and looks so much cleaner